### PR TITLE
Vungle Adapter Initial Commits

### PR DESCRIPTION
**Pull request by ashinagawa**
Adding adapter files - Vungle (compatible to version 5.1.0)

### CODE REVIEW

File: adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+@property (nonatomic, copy) NSString * _Nullable userId;
+
+/*!
+ * @brief Controls whether presented ads will start in a muted state or not.
+ */
+@property (nonatomic, assign) BOOL muted;
+
+
+@property (nonatomic, copy) NSArray<NSString *> * _Nonnull placements;
```
Comment (Rajaprahlad) [inline adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h]: Please use a more descriptive name for this property. You can use `allPlacements` similar to the [Android adapter](https://github.com/Vungle/googleads-mobile-android-mediation/blob/f65ea147a484fdf42c51a4c44b9f9d6bdb619bd0/ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java#L13).

File: adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+@property (nonatomic, copy) NSString * _Nonnull placement;
```
Comment (Rajaprahlad) [inline adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h]: Similarly, you can use `playingPlacement` here.

File: adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+- (void)dealloc {
+	[vungleHelper sharedInstance].interstitialDelegate = nil;
+}
+
+- (void)getBannerWithSize:(GADAdSize)adSize{
+	NSError *error = [NSError errorWithDomain:@"google"
+										 code:0
+									 userInfo:@{
+												NSLocalizedDescriptionKey : @"No ad fetched."
```
Comment (Rajaprahlad) [inline adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m]: Please update the error description to "Vungle doesn't support banner ads.".

File: adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h (hunk 1 above)
Comment (ashinagawa) [inline adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h]: Updated accordingly.

File: adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h (hunk 2 above)
Comment (ashinagawa) [inline adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h]: Updated accordingly.

**Pull request merged**